ocular/Cargo.toml

```diff
@@ -12,12 +12,23 @@ readme = "README.md"
 
 [dependencies]
 assay = "0.1"
+bip32 = "0.3.0"
 futures = { version = "0.3.19", features = ["executor"] }
 octocrab = "0.15.3"
 rand = "0.8.4"
 reqwest = { version = "0.11.8", features = ["json"] }
 serde = "1.0.133"
 serde_json = "1.0.78"
 tendermint-rpc = { version = "0.23.5", features = ["http-client"] }
+signatory = "0.24.0"
 thiserror = "1.0.30"
+tiny-bip39 = "0.8.2"
 url = "2.2.2"
+keyring = "1.1.2"
+k256 = { version = "0.10.2", features = ["pem"] }
+pkcs8 = "0.8.0"
+secp256k1 = "0.21.3"
```

Comment:
Why are you using both `k256` and `secp256k1`?

Reply:
Agreed this was unnecessary. Removed secp256k1 after some refractoring.